.io-config.json

```diff
@@ -0,0 +1 @@
+{"app_id":"b6a57699","api_key":"acbf10269e89184d103dcb94e36b989a24dfd84d5c4ffbac","dev_push":true}
```

Comment:
Also, this is in the `.gitignore`. Surprised that it even showed up in the diff.